appveyor.yml

```
@@ -50,9 +52,15 @@ install:
   - "powershell ./continuous_integration/appveyor/install.ps1"
   - "SET PATH=%PYTHON%;%PYTHON%\\Scripts;%PATH%"
   - "python -m pip install -U pip tox"
-  - "pip --version"
 
-  # Check that we have the expected version and architecture for Python
+  # next line only to demo that py3.8-on-Appveyor *could* install Cython from wheel just fine,
```

Adding capitalization/punctuation to make it easier to see where one statement ends and another begins.

```suggestion
  # Next line only to demo that py3.8-on-Appveyor *could* install Cython from wheel just fine,
```